Guidelet/GuideletLib/Guidelet.py

```diff
@@ -297,18 +306,41 @@ def registerCustomLayouts(self):#common
     self.triple3dCustomLayoutId=506
     layoutLogic.GetLayoutNode().AddLayoutDescription(self.triple3dCustomLayoutId, customLayout)
 
+  # -----------------------------------------------------
+  def onSceneLoaded(self):
+    # Derived classes can override this function
+    pass
 
+  # -----------------------------------------------------
   def setupScene(self):
+    '''
+    ReferenceToRas transform is used in almost all IGT applications. Reference is the coordinate system
```

Comment:
Guidelet does not need and does not know about ReferenceToRas. Please move this to UltraSound.setupScene().